### PR TITLE
Feature/#578 show only suitable utm zones using pypi packages

### LINKED ISSUE #578: Show only suitable UTM Zones
For the selected area, create a service, so that:

- [ ] sending an bounding box returns only the available UTM-Zones for that area
- [ ] using Javascript, the selectable list is being reduced to those choices

### PR DESCRIPTION
**Pull request by hixi**
Reviewed by:
- [x] @das-g
- [x] @hixi

Closes #578

### CODE REVIEW

File: osmaxx/excerptexport/static/excerptexport/scripts/utm_zone_filter.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                var srids = data["utm_zone_srids"].sort(),
+                    options_html = '';
+                srids.forEach(function(srid){
+                    var original_text = utm_zone_optgroup_original.find('option[value=' + srid + ']').text();
+                    options_html += _optionHTML(srid, original_text);
+                });
+                utm_zone_optgroup.html(options_html);
+            }).error(function(data, other, other2, other3){
+                console.log(data, other, other2, other3);
+            });
```
Comment (das-g) [inline osmaxx/excerptexport/static/excerptexport/scripts/utm_zone_filter.js]: `.error()` is deprecated, use `.fail(function( jqXHR, textStatus, errorThrown ) {})` or pass the function as the `error` argument to `jQuery.ajax(...)`.

File: osmaxx/excerptexport/static/excerptexport/scripts/utm_zone_filter.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        function getUTMZones(leafletGeometry) {
+            var geometry = leafletGeometry.toGeoJSON()["geometry"],
+                srid = 4326,
+                csrftoken = Cookies.get('csrftoken');
+            return jQuery.ajax({
+                url: '/api/utm-zone-info/',
+                contentType: "application/json; charset=utf-8",
+                dataType: 'json',
+                type: 'POST',
```
Comment (das-g) [inline osmaxx/excerptexport/static/excerptexport/scripts/utm_zone_filter.js]: As we are using jQuery &geq; 1.9 (currently, we're using 1.11), prefer `method: 'POST'` over `type: 'POST'`.

**Comment by das-g** [comment]
@hixi I've made (and have manually tested) some refactorings, please review.

File: osmaxx/excerptexport/static/excerptexport/scripts/utm_zone_filter.js (hunk 2 above)
Comment (hixi) [inline osmaxx/excerptexport/static/excerptexport/scripts/utm_zone_filter.js]: then let's use that.

**Pull request merged**